scipy/special/xsf/loggamma.h

```diff
@@ -85,13 +85,14 @@ namespace detail {
          * where gamma is the Euler-Mascheroni constant.
          */
 
-        double coeffs[] = {
-            -4.3478266053040259361E-2, 4.5454556293204669442E-2, -4.7619070330142227991E-2, 5.000004769810169364E-2,
-            -5.2631679379616660734E-2, 5.5555767627403611102E-2, -5.8823978658684582339E-2, 6.2500955141213040742E-2,
-            -6.6668705882420468033E-2, 7.1432946295361336059E-2, -7.6932516411352191473E-2, 8.3353840546109004025E-2,
-            -9.0954017145829042233E-2, 1.0009945751278180853E-1, -1.1133426586956469049E-1, 1.2550966952474304242E-1,
-            -1.4404989676884611812E-1, 1.6955717699740818995E-1, -2.0738555102867398527E-1, 2.7058080842778454788E-1,
-            -4.0068563438653142847E-1, 8.2246703342411321824E-1, -5.7721566490153286061E-1};
+        double coeffs[] = {-4.3478266053040259361E-2, 4.5454556293204669442E-2,  -4.7619070330142227991E-2,
```

Comment:
Why does clang-format move the first line of numbers into the same line as `{` here? I noticed that in some places clang-format leaves the numbers in their own line. Do you by chance know what is the rule?

Reply:
I'm not sure.

Reply:
Maybe `Cpp11BracedListStyle: true`